src/concat_impl.rs

```diff
@@ -1,8 +1,6 @@
-use crate::Itertools;
-
 /// Combine all an iterator's elements into one element by using [`Extend`].
 ///
-/// [`IntoIterator`]-enabled version of [`Itertools::concat`].
+/// [`IntoIterator`]-enabled version of [`crate::Itertools::concat`].
```

Comment:
Usually, we do ``[`Itertools::...`](crate::Itertools::...)``. It's longer but visually nicer (once rendered): no `crate::` seen.
Same twice in the "kmerge.." file.

Reply:
Ah good point, thanks :)